content/02.introduction.md

```diff
@@ -25,16 +26,28 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
 We will critique, sort and distill informative contents out of the overwhelming flood of information and help the larger scientific community to be better educated on this critical subject affecting all of us.
 This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.
 
-
 ### About Coronaviruses
 
-#### Classification:
+#### Coronavirus Genome and Structure
+
+Coronaviruses (CoVs; order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*) are enveloped viruses with some of the largest viral RNA genomes.
+Coronavirus genomes range from 27 to 32 kilobases (Kb) in length.
+The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
+There are several fundamental genomic characteristics shared by all viruses in the *Nidovirales* order, including enveloped, non-segmented, and positive-sense (ssRNA+) genomes and a large number of non-structural genes originating through ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
+Genome organization is highly conserved within the order [@doi:10.1007%2F978-1-4939-2438-7_1].
+Coronavirus genomes are comprised of a replicase gene that is two-thirds of the genome.
+This polypeptide is translated into 16 non-structural proteins (nsp1-16, except in Gammacoronaviruses, where nsp1 is absent), which form replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
+The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
+Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
+<!-- To do: Add a figure showing the structure of the genome.-->
+<!-- Also, everything focuses on S! Should we also talk about others?-->
```

Comment:
That definitely makes sense! It might be worth looping in @nilswellhausen who has offered to lead the charge on scientific visualizations -- if we include a diagram, what complementary information would be helpful to have in the text?

Reply:
I can make a more visually appealing version of this published genome which is very thorough from a scientific standpoint: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7161481/

Reply:
That sounds great. I think based on the information in the figure linked here, I think what we have in the text is sufficient..